### DISCUSSION TITLE
Unable to correctly download a video to tvaplus.ca

### DISCUSSION THREAD

**Discussion by JustBeta** [aclap-dev/video-downloadhelper#1197] [Help and Q&A]
### Your question

Hello,

I use the tvaplus.ca (qub.ca) platform and I'm sorry to say that video downloadhelper is unable to correctly download any video on this site.

I'm using the latest version (updated today) on a Windows Server 2022.

The vidéo downloaded is scrambled.

Thank you for your help.

### Addon & CoApp version and details

version: 9.1.0.43
target: microsoft
channel: stable
lang: fr
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36 Edg/128.0.0.0
{
    "used_history_button": true,
    "view_options": {
        "all_tabs": true,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 5,
    "successfull_dl": 7,
    "dable_attempts": {
        "__serializer_tag": "map",
        "__serializer_value": [
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "www.tvaplus.ca"
                    },
                    {
                        "__serializer_tag": "array",
                        "__serializer_value": [
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 11
                            },
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 7
                            }
                        ]
                    }
                ]
            }
        ]
    },
    "dable_attempts_last": 1725118522420,
    "record_download_history": true
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another utilisateur.

À chaque reprise, mes tentatives de faire jouer quoi que ce soit sur le site me donnent la suite:

Lecture interrompue
Une erreur inattendue s'est produite, veuillez réessayer plus tard.

J'imagine qu'il n'y a aucun temps qui est assez plus tard.  I would not be the slightest bit surprised if I am being geoblocked on this site.

Meanwhile, you say what you download is scrambled.  Sounds to me like they are protecting their content with DRM.  Search this forum for other discussions of DRM.  Use of DRM seems to be spreading, although my personal opinion is that it should not be so prevalent.  I suspect it's going to take a test case, a class action lawsuit, to get DRM removed from a lot of sites where it really shouldn't be applied.  Right.  Let's all hold our breath waiting for that . . .

**Comment by JustBeta** [comment]
Yes, it's exact, this site use DRM for streaming.

I didn't know that VDH doesn't support DRM.

**Reply by GrampaWildWilly** [reply]
DRM is meant to protect copyrighted material from copying.  This is not unique to VDH.  DRM blocks all downloads, no matter what tool you use.  I believe there are services, not applications, but web sites, that sell subscriptions that allow you to download DRM content.  I believe they are not cheap.  If you're that interested, do a web search.
